src/transport_picoquic.cpp

```diff
@@ -808,8 +827,10 @@ PicoQuicTransport::enqueue(const TransportContextId& context_id,
 
         if (stream_cnx != ctx->second.end()) {
             metrics.enqueued_objs++;
+            // add the app data to the queue
+            // mark the stream active if it isn't already
```

Comment:
Seems redundant to state, the code below seems well written and self explanatory.